src/hooks/job-schedules/use-job-schedules.js

```diff
@@ -0,0 +1,26 @@
+import { useDataQuery } from '@dhis2/app-runtime'
+
+const key = 'schedules'
+const query = {
+    [key]: {
+        resource: 'scheduler',
+    },
+}
+
+const useJobSchedules = () => {
+    const fetch = useDataQuery(query)
+
+    // Remove nesting from data and move the id up
+    if (fetch.data?.[key]?.map) {
+        const data = fetch.data[key].map((job) => {
+            const id = job.sequence?.[0]?.id
+            return { ...job, id }
+        })
+
+        return { ...fetch, data }
+    }
```

Comment:
Just a suggestion to make it easier to read

```suggestion
    const fetch = useDataQuery(query)
    const schedules = fetch.data?.schedules

    // Remove nesting from data and move the id up
    if (schedules?.map) {
        const data = schedules.map((job) => {
            const id = job.sequence?.[0]?.id
            return { ...job, id }
        })

        return { ...fetch, data }
    }
```

Reply:
I modified the hooks a little in [f53059b](https://github.com/dhis2/scheduler-app/pull/495/commits/f53059ba406181d3fce93f733bc6ae4249d73553). All hooks now have a condition that applies anytime fetch.data is truthy. Inside that if I've applied your suggestion wherever possible. Let me know if there's anything that should still be addressed.